fmc/src/hand_off.rs

```diff
@@ -216,22 +283,25 @@ impl HandOff {
     }
 
     /// The FMC CDI is stored in a 32-bit DataVault sticky register.
-    fn rt_cdi_store(output: &DiceOutput) -> HandOffDataHandle {
-        HandOffDataHandle(((Vault::KeyVault as u32) << 12) | output.cdi as u32)
+    fn rt_cdi_store(rt_cdi: KeyId) -> HandOffDataHandle {
+        HandOffDataHandle(((Vault::KeyVault as u32) << 12) | rt_cdi as u32)
     }
 
-    fn rt_priv_key_store(output: &DiceOutput) -> HandOffDataHandle {
-        HandOffDataHandle(((Vault::KeyVault as u32) << 12) | output.subj_key_pair.priv_key as u32)
+    fn rt_priv_key_store(rt_priv_key: KeyId) -> HandOffDataHandle {
+        HandOffDataHandle(((Vault::KeyVault as u32) << 12) | rt_priv_key as u32)
     }
```

Comment:
nit: This can be a generic function `get_keyid_handle` right? It looks like they're doing the same thing

Reply:
Created an issue as follow up